Gemfile

```diff
@@ -26,7 +26,7 @@ group :development, :test do
 end
 
 group :test do
-  gem 'brakeman', '~> 4.3.0', require: false
+  gem 'brakeman', '~> 4.4.0', require: false
```

Comment:
Any reason for bumping this as well? I think `brakeman` is one of those dependencies that should be kept unlocked actually – we want it to fail as it gets updates a new vuln comes in. Unrelated to this PR, so no blocker.